ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -288,6 +284,11 @@ export const WorkspaceEdit = fp.flow(withCurrentWorkspace(), withCdrVersions(),
       };
     }
 
+    // temporarily force all interactions to Registered Tier while we debug the old cdr modal issue
```

Comment:
I don't know why this resolves the issue.  But I've tested it locally.  I'll keep working on a real solution after we merge this to unblock the release.